newdle/api.py

```diff
@@ -29,3 +41,40 @@ def require_token():
 @api.route('/me/')
 def me():
     return UserSchema().jsonify(g.user)
+
+
+def _generate_fake_users():
+    f = Faker()
+    f.seed_instance(0)
+
+    def _generate_fake_user():
+        first_name = f.first_name()
+        last_name = f.last_name()
+        email = f'{first_name}.{last_name}@{f.domain_name()}'.lower()
+        return {'first_name': first_name, 'last_name': last_name, 'email': email}
+
+    return [_generate_fake_user() for _ in range(100)] + [
+        {
+            'first_name': g.user['first_name'],
+            'last_name': g.user['last_name'],
```

Comment:
I think `first_name` and `last_name` are bad naming. In Asia countries, their family name come at first.

Reply:
While our userbase is *mostly* western, that's a valid point - luckily our UI uses primarily the `name` for display purposes, which is currently built from first+last in the schema but could just as well be taken directly from whatever "name" the SSO provider sends (but I have the feeling over there it's also just concatenating first and last name, since at least in our case that data is coming from the HR database where I'm pretty sure it's already split into first and last name)